tools/engine_tool/lib/src/commands/test_command.dart

```diff
@@ -50,19 +50,29 @@ final class TestCommand extends CommandBase {
       return 1;
     }
 
-    final Map<String, TestTarget> allTargets = await findTestTargets(
-        environment,
+    final Map<String, BuildTarget> allTargets = await findTargets(environment,
         Directory(p.join(environment.engine.outDir.path, build.ninja.config)));
-    final Set<TestTarget> selectedTargets =
+    final Set<BuildTarget> selectedTargets =
         selectTargets(argResults!.rest, allTargets);
     if (selectedTargets.isEmpty) {
       environment.logger.error(
           'No build targets matched ${argResults!.rest}\nRun `et query tests` to see list of targets.');
```

Comment:
Is et query tests still right? A test only _target_ is not the same as a test (I think that's what you're fixing?)

Reply:
This text has now been updated to call out 'et query targets'